orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/utils/MonikerHelper.java

```diff
@@ -58,4 +59,20 @@ static public Moniker monikerFromStage(Stage stage) {
       return null;
     }
   }
+
+  static public Moniker monikerFromStage(Stage stage, String fallbackFriggaName) {
+    Moniker moniker = monikerFromStage(stage);
```

Comment:
Q: what happens when you have a stage that's dealing with potentially > 1 server group or cluster? 

`stage.context.moniker` seems to imply there only ever being one, and nothing in the context key indicates it's intent.

The rollback stages are a current example with > 1 server group specified.

Reply:
So in the case of TrafficGuard, the moniker:
```
"moniker" : {
      "app" : "nginx",
      "cluster" : "nginx-trafficguards",
      "detail" : null,
      "sequence" : 11,
      "stack" : "trafficguards"
    }
```
would be passed through. In TrafficGuard itself only `moniker.cluster` is used. In `ClusterMatcher` only `moniker.stack` and `moniker.detail` are used. `moniker.sequence` is not used. So the moniker is more of a cluster moniker than a server group moniker.

Reply:
@ajordens Do you know an example of a stage that uses multiple clusters?